chore(button): Added custom-elements representation for a button (#769)
dynatrace-oss/barista: libs/fluid-elements/button/README.md

> # barista-button

Reviewer: ```suggestion
# fluid-button
```

This is an experimental button element built with lit-elements and

> web-components. It registers itself as `barista-button` custom element.

Reviewer: ```suggestion
web-components. It registers itself as `fluid-button` custom element.
```

| `--barista-button--background-active`   | Controls the background color of the button in active mode.   |
| `--barista-button--background-disabled` | Controls the background color of the button in disabled mode. |
| `--barista-button--background-hover`    | Controls the background color of the button in hover mode.    |
| `--barista-button--border-radius`       | Controls the border radius of the button.                     |
| `--barista-button--border-style`        | Controls the border style of the button.                      |
| `--barista-button--border-width`        | Controls the border width of the button.                      |
| `--barista-button--foreground`          | Controls the default foreground color.                        |
| `--barista-button--foreground-active`   | Controls the foreground color of the button in active mode.   |
| `--barista-button--foreground-disabled` | Controls the foreground color of the button in disabled mode. |
| `--barista-button--foreground-hover`    | Controls the foreground color of the button in hover mode.    |
| `--barista-button--height`              | Controls the height of the button.                            |

> | `--barista-button--padding`             | Controls the inner padding of the button.                     |

Reviewer: ```suggestion
| `--fluid-button--background`          | Controls the default background color.                        |
| `--fluid-button--background-active`   | Controls the background color of the button in active mode.   |
| `--fluid-button--background-disabled` | Controls the background color of the button in disabled mode. |
| `--fluid-button--background-hover`    | Controls the background color of the button in hover mode.    |
| `--fluid-button--border-radius`       | Controls the border radius of the button.                     |
| `--fluid-button--border-style`        | Controls the border style of the button.                      |
| `--fluid-button--border-width`        | Controls the border width of the button.                      |
| `--fluid-button--foreground`          | Controls the default foreground color.                        |
| `--fluid-button--foreground-active`   | Controls the foreground color of the button in active mode.   |
| `--fluid-button--foreground-disabled` | Controls the foreground color of the button in disabled mode. |
| `--fluid-button--foreground-hover`    | Controls the foreground color of the button in hover mode.    |
| `--fluid-button--height`              | Controls the height of the button.                            |
| `--fluid-button--padding`             | Controls the inner padding of the button.                     |
```
